lifecycle-operator/apis/lifecycle/v1beta1/keptnworkloadversion_types.go

```diff
@@ -521,3 +521,13 @@ func (w KeptnWorkloadVersion) GetEventAnnotations() map[string]string {
 		"workloadVersionName": w.Name,
 	}
 }
+
+func (w *KeptnWorkloadVersion) SetDeploymentStartTime() {
+	if w.Status.DeploymentStartTime.IsZero() {
```

Comment:
nitpick: since we have this check implemented in its own function we can also use that here

```suggestion
	if !w.IsDeploymentStartTimeSet() {
```

Reply:
you are right!